webserver/__init__.py

```diff
@@ -35,12 +35,22 @@ def create_app(debug=None, config_path=None):
     from db import init_db_engine
     init_db_engine(app.config['SQLALCHEMY_DATABASE_URI'])
 
-    # Memcached
-    if 'MEMCACHED_SERVERS' in app.config:
-        from db import cache
-        cache.init(app.config['MEMCACHED_SERVERS'],
-                   app.config['MEMCACHED_NAMESPACE'],
-                   debug=1 if app.debug else 0)
+    # Cache 
+    if 'REDIS_HOST' in app.config and\
+       'REDIS_PORT' in app.config and\
+       'REDIS_NAMESPACE' in app.config and\
+       'REDIS_NS_VERSIONS_LOCATION' in app.config:
+        if not os.path.exists(app.config['REDIS_NS_VERSIONS_LOCATION']):
```

Comment:
should be try/except instead of exists. We're still on python 2 so can't use the `exist_ok` flag to makedirs